testcases/elichika_tests/canonicalizer/Continue.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import ast, gast
+from chainer_compiler.elichika.parser import canonicalizer
+from chainer_compiler.elichika.parser import utils
+from chainer_compiler.elichika.testtools import compare_ast
+
+
+class Continue(unittest.TestCase):
+    def setUp(self):
+        self.canonicalizer = canonicalizer.Canonicalizer()
+
+    def test_continue(self):
+        orig_code = """
+        x = 0
+        for i in range(10):
+            if i >= 5:
+                continue
+            x += 1
+        """
+        target_code = """
+        x = 0
+        for i in range(10):
+            continued_0 = False
+            if i >= 5:
```

Comment:
[optional] I'd slightly prefer `==` so the result will be different from the test for `break`. Maybe it'd be also better to change the test for `break` as well?